packages/cms/pages/auth/register/register.tsx

```diff
@@ -66,6 +104,7 @@ export const Register: React.FunctionComponent = () => {
                 <EuiFieldText
                   autoFocus
                   isInvalid={!!errors?.firstName}
+                  value={!!params?.invite ? form?.firstName : ''}
```

Comment:
Shouldn't this just be `form?.firstName` ? If there's an invite then the value will already be set to state. 

Also this most likely breaks normal registration. Please check.